tests/test_barebones_mpc/config_files/default_test_config.yaml

```diff
@@ -0,0 +1,30 @@
+# Norlab-MPPI barebones_mpc config
+---
+config-name: 'default'
+config-short-description: ""
+controler-component:
+  model: src.barebones_mpc.model.abstract_model.MockModel
+  sampler: src.barebones_mpc.sampler.abstract_sampler.MockSampler
+  evaluator: src.barebones_mpc.evaluator.abstract_evaluator.MockEvaluator
+  selector: src.barebones_mpc.selector.abstract_selector.MockSelector
+hparam:
+  experimental-hparam:
+    horizon: 15
```

Comment:
Is the horizon defined in time or number of time steps? I like the approach of defining time step and time horizon in seconds. Then define another variable for sample length per prediction. The sample length is floor(horizon/time_step). See test_barebones_mpc_inv_pend_model.py at line 11 for an example.

Reply:
Ok we'll need a discussion on this and we'll need to choose an explicit word. In RL and MDP, timestep is a general concept related to advancing in the sequence. It can mean 1 millisecond, 1 hour, stepping to the next event ... It's usualy discreet but can be continous

Reply:
Also, I was thinking of `experimental-hparam/horizon` as a hparam for the ModelPredictiveController feedback loop execution 
```
experimental-hparam:
    horizon: 15
```
and of `sampler-hparam/sample_length` for the horizon of the `sampler` sampling roolout 
```
sampler-hparam:
    number_samples: 15
    sample_length: 50
```
Even do it's both timesteeping related, I think it's less error prone for us if we choose two different term for those two. Could be some other name